readme.txt

```diff
@@ -56,15 +56,23 @@ Per the primary purpose of the plugin (see above), it can mostly be considered a
 
 = Where can I submit my plugin feedback? =
 
-Especially since this is a collection of WordPress core feature plugins, providing feedback is encouraged and much appreciated! You can submit your feedback either in the [plugin support forum](https://wordpress.org/support/plugin/performance-lab/) or, if you have a specific issue to report, in its [GitHub repository](https://github.com/WordPress/performance).
+Feedback is encouraged and much appreciated, especially since this plugin is a collection of future WordPress core features. If you have suggestions or requests for new features, you can [submit them as an issue in the Performance Lab GitHub repository](https://github.com/WordPress/performance/issues/new/choose). If you need help with troubleshooting or have a question about the plugin, please [create a new topic on our support forum](https://wordpress.org/support/plugin/performance-lab/#new-topic-0).
 
 = How can I contribute to the plugin? =
 
-Contributions welcome! There are several ways to contribute:
+Contributions are always welcome! Learn more about how to get involved in the [Core Performance Team Handbook](https://make.wordpress.org/performance/handbook/get-involved/).
 
-* Raise an issue or submit a pull request in the [Github repository for the plugin](https://github.com/WordPress/performance)
-* Translate the plugin into your language at [translate.wordpress.org](https://translate.wordpress.org/projects/wp-plugins/performance-lab)
-* Join the weekly chat (Tuesdays at 16:00 UTC) in the [#performance channel on Slack](https://wordpress.slack.com/archives/performance)
+= I've activated the WebP Uploads module, but WebP images are not always generated when I upload a JPEG image. Why? =
+
+There are two primary reasons that a WebP image may not be generated:
+
+1. Performance Lab has identified that the WebP version of the uploaded JPEG image would have a larger file size than the original JPEG image, so it does not generate the WebP version.
+
+2. The JPEG image was not uploaded to the [Media Library](https://wordpress.com/support/media/). At this time, WebP versions are only generated for images to the Media Library. WebP versions are not generated for JPEG images that are added to your site in other ways, such as in a template file or the [Customizer](https://wordpress.com/support/customizer/).
+
+= With the WebP Uploads module activated, will Performance Lab generate JPEG and WebP versions of every image that I upload? =
+
+By default, the WebP Uploads module will only generate WebP versions of the images that you upload. If you wish to have both WebP **and** JPEG versions generated, you can navigate to **Settings > Media** and enable the **Generate JPEG files in addition to WebP** option.
```

Comment:
Should include steps for multisite (Once 1.7.0 out). 

Something like:
**Navigate to Top left My Sites menu > Select your site > Settings > Media**

Reply:
@mukeshpanchal27 Hm, I would disagree here. Generally on Site Kit documentation, for example, we provide the instructions for single sites only unless the multisite instructions are in a completely different place. In this case, the owner of a multisite should know to go into their own single site to change the setting there. @felixarntz WDYT?

Reply:
@bethanylang I guess, we still need to mention here that it is in subsite settings, as it can be confusing for users to go network admin and Settings > Media or in sub site admin.

Reply:
@bethanylang Agreed, for multisites it's implied that settings work the same way like in a single site, except when they don't (but here they do). So IMO we don't need to call out multisite here. The checkbox has the same content and shows on the same page for multisites.

Reply:
Thanks Felix.